File: hammer-chat/src/main/java/dev/vini2003/hammer/chat/HCClient.java
package dev.vini2003.hammer.chat;

import dev.vini2003.hammer.chat.registry.client.HCEvents;
import dev.vini2003.hammer.chat.registry.client.HCNetworking;
import net.fabricmc.api.ClientModInitializer;
import org.jetbrains.annotations.ApiStatus;

@ApiStatus.Internal
public class HCClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		HCEvents.init();
		HCNetworking.init();
	}
}

File: hammer-chat/src/main/kotlin/dev/vini2003/hammer/chat/HCClient.kt
package dev.vini2003.hammer.chat

import dev.vini2003.hammer.chat.registry.client.HCCommands
import dev.vini2003.hammer.chat.registry.client.HCNetworking

object HCClient {
	fun init() {
		HCCommands.init()
		HCNetworking.init()
	}
}